webnn.json

```diff
@@ -15,6 +15,32 @@
     "See the License for the specific language governing permissions and",
     "limitations under the License."
   ],
+  "instance descriptor": {
+    "category": "structure",
+    "members": []
+  },
+  "instance": {
+    "category": "object",
+    "methods": [
```

Comment:
just one question, you mentioned other classes like NamedOperands will be managed by instance, how? And will they be created by instance?

Reply:
checked with @fujunwei , other classes will be managed by instance in following PRs.